analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConstructorArgumentValueShouldExist.CSharp9.cs

```diff
@@ -11,3 +11,20 @@ public class MyExtension3 : MarkupExtension
 
     public override object ProvideValue(IServiceProvider serviceProvider) => null;
 }
+
+// This is fake scaffolding to avoid .NET 5 reference issue.
```

Comment:
This looks to be a fix which can be reused. What do you think about moving it to the project generation part?

Reply:
It's just this one rule that needs these specific classes. I would keep this trick only in this rule. We have similar ones elsewhere for `System.Windows`:
https://github.com/SonarSource/sonar-dotnet/blob/ebcf626d64d8d2bbdbfd9f95db50697113e90c44/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ReferenceEqualityCheckWhenEqualsExists.cs#L6-L24

We can build something better in case we'll have a problem with it (where `problem` could be a WPF-related MMF 😃 )